policies/CIS.2.7.rego

```diff
@@ -0,0 +1,16 @@
+package cis_2_7
+
+import data.lib.kubernetes
+
+params = {
+    "key": "--client-ca-file", 
+    "key2": "--etcd-cafile"
```

Comment:
Perhaps if we can rename key and key2 to something else, I just used those for testing.

Maybe `apiserverCaKey` and `etcdCaKey`?